spec/filters/dissect_spec.rb

```diff
@@ -3,6 +3,31 @@
 require "logstash/filters/dissect"
 
 describe LogStash::Filters::Dissect do
+  class LoggerMock
```

Comment:
Will do.
I am hoping that @talevy work with log4j2 in JRuby/Java will end up with a Cabin Channel facade that allows Java plugins etc to log in Java and that the log entries end up in the logs Array because I want to be able to use his work in this plugin eventually.